.goreleaser.yml

```diff
@@ -140,3 +142,93 @@ changelog:
       order: 30
     - title: Other
       order: 999
+
+# https://goreleaser.com/customization/homebrew/
+brews:
+  - name: observiq-otel-collector
+    tap:
+      owner: observIQ
+      name: homebrew-observiq-otel-collector
+      branch: main
+    download_strategy: CurlDownloadStrategy
+    folder: Formula
+    url_template: https://github.com/observIQ/observiq-otel-collector/releases/download/{{ .Tag }}/{{ .ArtifactName }}
+    commit_author:
+      name: observiq
+      email: support@observiq.com
+    homepage: "https://github.com/observIQ/observiq-otel-collector"
+    license: "Apache 2.0"
+    caveats: |
+      ****************************************************************
+      After installation please run the following to finish setup:
+
+        # Default location for the opentelemetry-java-contrib-jmx-metrics.jar
```

Comment:
Should probably note that this is only required if monitoring JVM technologies with JMX metrics.

Reply:
Yeah I did so in the actual documentation. I didn't want to overload the homebrew prompt. Still think it should be in the prompt?

Reply:
Remove from prompt works for me.